Cargo.toml

```diff
@@ -28,9 +28,9 @@ log = "0.4"
 either = "1.5"
 unicode-segmentation = "1.2"
 unicode-width = "0.1"
-termion = { version = "1.5", optional = true }
+termion = { version = "1.0" , optional = true }
```

Comment:
why a change on the termion version ?

Reply:
Ah, good catch, I was trying something out there. But should be reverted.